python/tskit/trees.py

```diff
@@ -5442,3 +5442,50 @@ def newick_trees(self, precision=3, breakpoints=None, Ne=1):
             "This method is no longer supported. Please use the Tree.newick"
             " method instead"
         )
+
+    def nexus(self, precision=14, node_labels=None):
+        """
+        Returns a nexus encoding <https://en.wikipedia.org/wiki/Nexus_file>`_
+        of this tree.
+        :param int precision: The numerical precision with which branch lengths are
+            printed.
+        :param dict node_labels: If specified, show custom labels for the nodes
```

Comment:
Let's leave this parameter out, for now. I think we keep the newick strings having their Node IDs as labels. We can add in support for the TRANSLATE blocks in a follow up.